### PR TITLE
Fix #5508: remove min on diffractionOrder

### LINKED ISSUE #5508: Grating diffraction order needs to allow negative integers
> In "Grating -> Orientation" panel, the "Diffraction Order" input box doesn't seem to allow for entering negative order numbers (-1, -2, etc.), however the use of gratings in this orientation is quite common. I don't know at this point, maybe some tests and fixes will be required in this relation in SRW; but Sirepo should allow entering negative orders.
> One could add validation that user's input in this dialog is limited to integer numbers only, but this is less important.

### PR DESCRIPTION
**Pull request by gurhar1133**
Negative diffraction order allowed now

### CODE REVIEW

**Review by moellep — CHANGES_REQUESTED** [review]
Also, change the field type from Float to Integer.

**Review by moellep — APPROVED** [review]

File: tests/template/srw_shadow_data/to_shadow_grating.out/shadow.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
             "offx": 0,
             "offy": 0,
             "offz": 0,
-            "order": -1.0,
+            "order": 1,
             "param": 0.0,
```
Comment (moellep) [inline tests/template/srw_shadow_data/to_shadow_grating.out/shadow.json:87]: This test case is a little odd that it changed sign from -1.0 to 1. Could you look at the conversion in srw_shadow.py and make sure it is doing the right thing?

Comment (gurhar1133) [reply tests/template/srw_shadow_data/to_shadow_grating.out/shadow.json:87]: In `srw_shadow.py`, if the type for the field is Float in the schema then the value will be multiplied by `scale` which is -1 for diffractionOrder:

```python
if schema[from_f][1] == "Float":
        v = float(v) * scale
```

But the type was changed to Integer. Do we still want to scale by -1 now that it is integer?

Comment (moellep) [reply tests/template/srw_shadow_data/to_shadow_grating.out/shadow.json:87]: Yes, I think that should be changed to check if it is Float or Integer, apply the scale. Shadow needs the diffraction order to be negative to agree with SRW.

**Pull request merged**